Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -32,7 +68,38 @@ class CustomerInfoResponseHandler {
             }
             .mapError(BackendError.networkError)
 
-        completion(result)
+        self.handle(result: result, completion: completion)
+    }
+
+    private func handle(
+        result: Result<CustomerInfo, BackendError>,
+        completion: @escaping CustomerAPI.CustomerInfoResponseHandler
+    ) {
+        guard self.offlineEntitlementsEnabled,
+              result.error?.isServerDown == true,
```

Comment:
I believe we are missing another condition here for when to calculate offline entitlements... When performing a "get customer info" request, we should only calculate offline entitlements if there is no cached customer info in disk. Otherwise, we should just rely on the cached customer info. Are you handling that somewhere else?